.github/workflows/c-cpp.yml

```
@@ -6,10 +6,23 @@ on:
+env:
+  __bin: dbs26
+  __c_src: args.c dbs26.c
+  __builddir: ${{ github.workspace }}/build
+  __msvc_git: ${{ github.workspace }}/msvc
+  __srcdir: ${{ github.workspace }}/src
+
 jobs:
   build-ubuntu-24_04:
     runs-on: ubuntu-24.04
     steps:
+    - env:
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)